components/date-picker/date-picker.component.ts

```diff
@@ -135,6 +135,9 @@ export class NzDatePickerComponent implements OnInit, OnChanges, OnDestroy, Cont
   @Input() @InputBoolean() nzAutoFocus: boolean = false;
   @Input() @InputBoolean() nzDisabled: boolean = false;
   @Input() @InputBoolean() nzInputReadOnly: boolean = false;
+  /**
+   * @deprecated when datePanel component complete
+   */
```

Comment:
```suggestion
  /**
   * @deprecated use method `open` instead.
   * @breaking-change 11.0.0
   */
```

And call `warnDeprecation` in `ngOnChanges`.